src/components/quick-delegate-banner/quick-delegate-banner.tsx

```diff
@@ -31,7 +33,7 @@ export const QuickDelegateBannerComponent = (
 
     const blockchainConfig = getBlockchain(props.blockchain);
 
-    if (!props.account) {
+    if (!props.account && !props.accountStats?.chartStats) {
```

Comment:
is there any dependency on chartStats for this banner ?

Reply:
not anymore

Reply:
ok, remove it :)